src/run.ts

```diff
@@ -104,6 +78,44 @@ async function downloadHelm(version: string): Promise<string> {
     return helmpath;
 }
 
+async function getLatestHelmVersionFor(type) {
+    const token = core.getInput('token', { 'required': true });
+    try {
+        const { repository } = await graphql(
+            `{
```

Comment:
Indent this?

Reply:
I used the auto formatting tool in vs code. Probably because of (`) it doesn't appear properly indented. Anyway, I'll run the formatter again.